packages/components/package.json

```diff
@@ -29,7 +29,9 @@
     "react-markdown": "^4.3.1",
     "react-modal": "^3.11.2",
     "react-tooltip": "^4.2.6",
-    "rimraf": "^3.0.2"
+    "rimraf": "^3.0.2",
+    "styled-components": "^5.1.1",
+    "twin.macro": "^1.4.1"
```

Comment:
Are we sure about that? Are we planning to use styled-components for styles from now on? @tkanzakic @stevejcox 

Adding three libraries to be able to change backgroundColor of tooltip seems overwhelming .

We might think a different approach to solve the original issue. Or we might commit to start working with css in js and use styled-components everywhere.

Reply:
Chatted to @dawinValenzuela - going to remove styled components for now. I agree that if we want to go down that route, we should refactor to use that everywhere. And there are plenty of pros and cons for that approach, so we'd want to regroup about it first.